build.gradle.kts

```diff
@@ -590,6 +587,16 @@ allprojects {
                     }
                 }
             }
+
+            register<Test>("testSlow") {
+                group = LifecycleBasePlugin.VERIFICATION_GROUP
+                description = "Runs the slow unit tests."
+                useJUnitPlatform() {
+                    includeTags("slow")
+                }
+                jvmArgs("-Xmx6g")
+                jvmArgs("-Djdk.net.URLClassPath.disableClassPathURLCheck=true")
```

Comment:
`-Djdk.net.URLClassPath` will be set by `withType<Test>().configureEach` above

Reply:
Actually, I was not sure where is the place to add new tasks. Is it fine to do it here?

Reply:
This is OK, however, I would move it closer to `withType<Test>` (before it or after it) just to keep the related things together.

It would add `testSlow` task to all the projects that have tests with `slow` tag.
Alternative option is to add new folders like `src/slowTests/java/...` (it is typically used for `src/integTests/...`, `src/jmh/...`), but I think `slow` tag is ok.